test/python/circuit/classical/test_expr_constructors.py

```diff
@@ -68,15 +68,27 @@ def test_value_lifts_qiskit_scalars(self):
 
     def test_value_lifts_python_builtins(self):
         self.assertEqual(expr.lift(True), expr.Value(True, types.Bool()))
+        self.assertEqual(expr.lift(True, try_const=True), expr.Value(True, types.Bool(const=True)))
```

Comment:
Just looking through the tests, I'm wondering if a future extension will be to add thread-local context variables for the `expr` constructors to allow contextual control over `try_const`, just because I'm vaguely thinking that it's slightly tricksy that the default user-facing `lift` is `try_const=False`, but the binary constructors use `try_const=True` in more places.

I think that's all fine for now, just trying to record for posterity, since I imagine we'll come back to look at the context for the `const` inclusion at some point in the future.

Reply:
No longer applicable with `Expr.const`.